functions/endChat.ts

```diff
@@ -9,10 +11,12 @@ import {
   functionValidator as TokenValidator,
 } from '@tech-matters/serverless-helpers';
 import type { TaskInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/task';
+import { ChatChannelJanitor } from './helpers/chatChannelJanitor.private';
 
 type EnvVars = {
   CHAT_SERVICE_SID: string;
   TWILIO_WORKSPACE_SID: string;
+  FLEX_PROXY_SERVICE_SID: string;
```

Comment:
Is this being used?

Reply:
context in line 94 requires this envvars to delete the proxy session - 
`await chatChannelJanitor(context, { channelSid });`